app/graphql/types/adjusted_fees/create_input.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Types
+  module AdjustedFees
+    class CreateInput < Types::BaseInputObject
+      description 'Create Adjusted Fee Input'
+
+      argument :fee_id, ID, required: true
+      argument :units, GraphQL::Types::Float, required: true
+      argument :unit_amount_cents, GraphQL::Types::BigInt, required: false
+      argument :invoice_display_name, String, required: false
```

Comment:
GraphQL/OrderedArguments: Arguments should be sorted in an alphabetical order within their section. Field `invoice_display_name` should appear before `unit_amount_cents`.